graphspace/settings/local.py

```diff
@@ -40,9 +40,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'test_database',
-        'USER': 'adb',
-        'PASSWORD': '',
+        'NAME': 'yash',
```

Comment:
@yash170106065 Can you remove the local.py from PR?

Reply:
ok i will remove it

Reply:
![Screenshot from 2020-01-23 04-06-15](https://user-images.githubusercontent.com/35887250/72940905-c2739e80-3d95-11ea-9209-4067ba925f00.png)

Reply:
it takes equal time approx for large no. of nodes also

Reply:
I meant include the exported pdf files. I looked at the implementation and it seems you are simply including the graph as a jpeg file in the pdf. I want to checkout the resolution of the exported files.

Also, on the side note --- checkout this [extension](https://github.com/kinimesi/cytoscape-svg) that exports current graph as an SVG image. Here is a simple [demo](https://kinimesi.github.io/cytoscape-svg).

If the resolution is not upto the mark you can first convert to SVG and then export as pdf.

Reply:
[https://drive.google.com/open?id=1gQ-zqW77maSLOIQGmuk1Ti-d9XoPqbGM](url)
[https://drive.google.com/open?id=1b5Al9ew6DB9_eKR6d8BKWIBQc4SI-SDi](url)

Reply:
https://drive.google.com/open?id=1gQ-zqW77maSLOIQGmuk1Ti-d9XoPqbGM

Reply:
https://drive.google.com/open?id=1b5Al9ew6DB9_eKR6d8BKWIBQc4SI-SDi

Reply:
@yash170106065 - I agree with @adbharadwaj on using SVG. Using vector image is preferable when exporting to PDF, this takes care of scaling issues of raster images formats like .jpeg, .png

Reply:
Ok I will do it in that way

Reply:
Glad you are looking into SVG conversion.

Reply:
You can also look at https://github.com/rainwoodman/nxsvg as an alternative since we manage the graphs as networkx object at the backend.

Reply:
Thank you so much @adbharadwaj  @jahandaniyal for guiding me I will try to do this on Saturday Sunday